src/asset-relocator.js

```diff
@@ -462,25 +463,55 @@ module.exports = async function (content) {
       // require
       else if (!isESM && isRequire(node)) {
         const expression = node.arguments[0];
-        // require('bindings')('asdf')
         if (isStaticRequire(node) &&
             parent.type === 'CallExpression' &&
             parent.callee === node &&
-            expression &&
-            expression.value === 'bindings') {
-          let staticValue = computeStaticValue(parent.arguments[0], true);
-          let bindingsValue;
-          if (staticValue && 'value' in staticValue) {
-            try {
-              bindingsValue = createBindings()(staticValue.value);
+            expression) {
+          // require('bindings')('asdf')
+          if (expression.value === 'bindings') {
+            let staticValue = computeStaticValue(parent.arguments[0], true);
+            let bindingsValue;
+            if (staticValue && 'value' in staticValue) {
+              try {
+                bindingsValue = createBindings()(staticValue.value);
+              }
+              catch (err) {}
+            }
+            if (bindingsValue) {
+              staticChildValue = { value: bindingsValue };
+              staticChildNode = parent;
+              staticChildValueBindingsInstance = staticBindingsInstance;
+              return this.skip();
             }
-            catch (err) {}
           }
-          if (bindingsValue) {
-            staticChildValue = { value: bindingsValue };
-            staticChildNode = parent;
-            staticChildValueBindingsInstance = staticBindingsInstance;
-            return this.skip();
+          // require('pkginfo')(module, ...string[])
+          else if (expression.value === 'pkginfo' &&
+                  parent.arguments.length &&
+                  parent.arguments[0].type === 'Identifier' &&
+                  parent.arguments[0].name === 'module') {
+            let filterValues = [];
+            for (let i = 1; i < parent.arguments.length; i++) {
+              if (parent.arguments[i].type === 'Literal')
+                filterValues.push(parent.arguments[i].value);
+            }
+            const scope = getPackageScope(id);
+            if (scope) {
+              try {
+                var pkg = JSON.parse(readFileSync(scope + '/package.json'));
+                if (filterValues.length) {
+                  for (var p in pkg) {
+                    if (filterValues.indexOf(p) === -1)
```

Comment:
This is going to be slow if there are a lot of filterValues or pkg keys.

Let's change `filterValues` to use a Map.

Reply:
Sure, I've updated to use a Set.

Reply:
Perfect! 👌